contracts/rollup/Rollup.sol

```diff
@@ -13,29 +13,82 @@ import { Create2Transfer } from "../Create2Transfer.sol";
 import { BatchManager } from "./BatchManager.sol";
 import { IDepositManager } from "../DepositManager.sol";
 
-contract RollupCore is BatchManager {
+contract Rollup is BatchManager {
```

Comment:
We no longer want to separate the Rollup and RollupCore contracts now. We didn't write tests that need to override values from RollupCore.